spec/views/post_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe 'Post views', type: :feature do
+  before(:each) do
+    @user1 = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+    @post = @user1.posts.create(title: 'This is my first post', text: 'This is the content of my first post')
+    @user1.posts.create(title: 'This is my second post', text: 'This is the content of my second post')
+    @user2 = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo1', bio: 'Teacher from Poland')
+    @comment = @post.comments.create(text: 'This is the first comment', author_id: @user2.id)
+  end
+
+  describe 'post show page' do
```

Comment:
- [ ] As requested by the previous reviewer, kindly complete writing feature tests for the post index page. Tests units were added only for the post show page.
- [ ] Also kindly make sure your testing file structure is organised. Some of your feature tests are found in the model folder instead of the view folder.